content/apis.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'The GDL Open APIs'
+---
+
+<section>
+
+# The GDL Open APIs
+
+The GDL platform provides a set of APIs, allowing for individual programmers, projects and organizations to build new apps and service with content from the Global Digital Library. We currently provide APIs for images, books and games. Each API listing offers documentation on the usage of the API plus the specification of operations.
+
+All APIs are open and available for anyone to use. 
+We have enable rate limiting on all our APIs. If you intend to use any of our APIs permanently, please [contact us](mailto:support@digitallibrary.io).
+Our APIs generally respond with JSON.
+
+<bottomimagewrapper>
+
+![feed image](/images/billy.png)
+
+</bottomimagewrapper>
+
+</section>
+
+<content>
+
+## Image API
+[Swagger](/image-api) &nbsp;&nbsp;&nbsp; [GitHub](https://github.com/GlobalDigitalLibraryio/image-api)
```

Comment:
@kievu Usikker på om `&nbsp;` er beste måten å oppnå det jeg vil på her. Vil ha to lenker, med litt luft mellom uten å måtte endre hele verden og style for å få det til...

Reply:
Det synes jeg er innenfor!

Spørs om disse burde heller være representert som knapper istedenfor lenker https://puu.sh/D8uEI/9338e28cda.png ?

Reply:
Synes knappene blir for svære og klumpete når det blir så mange av dem.